### DISCUSSION TITLE
Need to add welcome splash screen and car info for audi q2

### DISCUSSION THREAD

**Discussion by fuadamin** [Mr-MIBonk/M.I.B._More-Incredible-Bash#510] [Q&A]
Hi, 
I need add welcome splash screen and set car info for audi q2 with m.i.b._more incredible bash for my car. To do so, what should I do. Help is needed. Tia
![20240715_195457](https://github.com/user-attachments/assets/3fb9b9d7-05a9-449c-856d-d07da0ecd8ae)

![Screenshot_20240715_224319_Chrome](https://github.com/user-attachments/assets/4bd9796c-7510-46c5-ae92-a108a466bb30)
This screen stop to display after all in one patch is done.

**Comment by fuadamin** [comment]
The first problem is solved by using  mqbcoding(MIB toolbox). To do that, use model_db from mods folder and find the code for audi q2 (01 73 06)
01 - manufacturing company, 01 is audi.
73 - here 7 is car generation and 3 is car class.
![20240716_173150](https://github.com/user-attachments/assets/2d748eea-f045-477f-bf81-51c1781da0b5)

06 - here 0 is car derivate and 6 is car derivate supplement .
![20240716_173159](https://github.com/user-attachments/assets/c8136c86-280e-4a9e-adfd-c8ebccd345b9) 
** do not follow the picture just follow the text**
##############
Still looking for the solution for splashscreen display.

Tia

**Reply by cavaliersa** [reply]
Were you able to resolve this?
